src/server/search/search_family.cc

```diff
@@ -110,6 +117,47 @@ optional<SearchParams> ParseSearchParamsOrReply(CmdArgList args, ConnectionConte
       continue;
     }
 
+    // RETURN {num} [{ident} AS {name}...]
```

Comment:
This function is getting a little big, and it's only gonna get bigger as we add more options/paremeter(https://redis.io/commands/ft.search/). You don't have to do this now but for each optional parameter of `FT.search` consider adding a function and call it in the body of the for-loop.

Reply:
This is why I'm working on #1747. I will refactor everything here once both are merged